src/registrar/tests/test_emails.py

```diff
@@ -61,11 +61,27 @@ def test_disable_email_flag(self):
         # Assert that an email wasn't sent
         self.assertFalse(self.mock_client.send_email.called)
 
+    @boto3_mocking.patching
+    def test_email_with_cc(self):
+        """Test sending email with cc works"""
+        with boto3_mocking.clients.handler_for("sesv2", self.mock_client_class):
+            send_templated_email(
+                "emails/update_to_approved_domain.txt",
+                "emails/update_to_approved_domain_subject.txt",
+                "doesnotexist@igorville.com",
+                context={"domain": "test", "user": "test", "date": 1, "changes": "test"},
+                bcc_address=None,
+                cc_addresses=["testy2@town.com", "mayor@igorville.gov"],
+            )
+
+        # check that an email was sent
+        self.assertTrue(self.mock_client.send_email.called)
+
     @boto3_mocking.patching
     @less_console_noise_decorator
     def test_submission_confirmation(self):
         """Submission confirmation email works."""
-        domain_request = completed_domain_request()
+        domain_request = completed_domain_request(user=User.objects.create(username="test", email="testy@town.com"))
```

Comment:
By default the completed_domain_request function creates a user with no email. Technically what was happening previously was that emails were being "sent" (the send_email function was being called) but without any to_address because when the domain request was submitted, it would grab an empty string from the user's email. That was a valid case previously (not sure if that was intentional) but now I added a check to make sure that there is **some** email address in to, bcc, or cc. As a result, these tests broke and to fix it I had to add a user object with an email.